src/test/spec/unified_runner/entity.rs

```diff
@@ -44,14 +45,21 @@ pub struct ClientEntity {
     observe_sensitive_commands: bool,
 }
 
+impl ClientEntity {
```

Comment:
I don't think this method is necessary -- `ClientEntity` implements `Deref` with a target type of `Client` so you should be able to call `watch` directly on a `ClientEntity` (where this method is used).

Reply:
I missed that, thank you!  Removed.